gbpservice/nfp/service_vendor_agents/vyos/oc_config_server/server2.py

```diff
@@ -34,6 +34,7 @@
 from ha_config import VYOSHAConfig
 from vyos_exception import OCException
 from flask import jsonify
+from visibility_api import APIHandler as apihandler
```

Comment:
We should not directly import this. Let us make it pluggable. When visibility is not installed/non-commercial version, things should work.